Userland/Libraries/LibCore/Stream.h

```diff
@@ -671,11 +671,11 @@ class BufferedHelper {
         if (!stream().is_open())
             return Error::from_errno(ENOTCONN);
 
+        auto const candidate = TRY(find_and_populate_until_any_of(candidates, buffer.size()));
+
         if (buffer.is_empty())
             return Error::from_errno(ENOBUFS);
 
-        auto const candidate = TRY(find_and_populate_until_any_of(candidates, buffer.size()));
-
```

Comment:
This looks like only the updated commit message made it in, but not the actual change.

Reply:
Oops, I also removed another occurrence in `BufferedHelper::read()`